src/AutoMapper/QueryableExtensions/Impl/QueryMapperHelper.cs

```diff
@@ -15,10 +15,17 @@ public static PropertyMap GetPropertyMap(this IConfigurationProvider config, Mem
                                       pm.SourceMember != null && pm.SourceMember.Name == sourceMemberInfo.Name);
 
             if (propertyMap == null)
-            {
-                var message = $"Missing property map from {sourceMemberInfo.DeclaringType.Name} to {destinationMemberType.Name} for {sourceMemberInfo.Name} property. Create using Mapper.CreateMap<{sourceMemberInfo.DeclaringType.Name}, {destinationMemberType.Name}>.";
-                throw new InvalidOperationException(message);
-            }
+                throw new AutoMapperConfigurationException(new AutoMapperConfigurationException.TypeMapConfigErrors[] { new AutoMapperConfigurationException.TypeMapConfigErrors(typeMap, new string[] { sourceMemberInfo.Name }, true) });
```

Comment:
I think the constructor call needs to be reused below.

Reply:
which constructor call? below?????.  Please be clear.

Reply:
The same code appears [here](https://github.com/AutoMapper/AutoMapper/pull/2323/files#diff-31f5e95b00518c1126cd29aef4b975f3R27).

Reply:
Done